reference/prometheus.md

```diff
@@ -2,6 +2,16 @@ To see how to deploy a monitoring stack in Anbox Cloud, refer to the instruction
 
 The base installation provides basic dashboards. You can however update them to fit your needs. Below is the list of all metrics returned to Prometheus by each component of Anbox Cloud.
 
+## Endpoints
+
+The [monitoring stack](https://discourse.ubuntu.com/t/example-collect-metrics/17787) provides a starting point to monitor your environment.
+If you want to implement your own monitoring solution, you will find the API endpoints for every service that supports Prometheus metrics in the following table.
+
+| Service              | Endpoint                                           |
+|----------------------|----------------------------------------------------|
+| ams                  | `https://ams.example.com/internal/1.0/metrics`     |
+| anbox-stream-gateway | `https://gateway.example.com/internal/1.0/metrics` |
+| lxd                  | `https://lxd.example.com/1.0/metrics`              |
```

Comment:
The endpoints are not right, By default AMS and the gateway us different HTTP endpoints on other ports than their default ones. We should explain that we show the endpoint from an internal perspective and add the default ports next to a comment that the ports might be different if changed in the Juju configuration.